systers_portal/templates/community/request_community.html

```diff
@@ -9,7 +9,7 @@
   <div class="mt20 mb40"></div>
   <div class="row ml20">
     <div class="col-md-12">
-      <h1>Request a new community</h1>
+      <h1 class="request_form-head">Request a new community</h1>
```

Comment:
why do we have this strange casing ? ` request-form-head` please.

Reply:
@tonythomas01 that was to selectively edit the headings under h1

Reply:
to selectively edit, why should the casing of class attr matter ?

Reply:
@abhi20161997 I agree with @tonythomas01 , please change it to `request-form-head`